WooCommerce/WooCommerceUITests/Tests/OrdersTests.swift

```diff
@@ -32,6 +32,7 @@ final class OrdersTests: XCTestCase {
             .startOrderCreation()
             .editOrderStatus()
             .addProduct(byName: products[0].name)
+            .addCustomerDetails()
```

Comment:
On one hand, I do understand that making the test assert the presence of `Mira` on the resulting order page is a bit artificial, because this info is really specified in the `orders_3337.json`.

On the other hand, I still see value in this, because it will check the ability of UI to display this particular information from the response, no matter if this is real API or mocked.

What do you think?